usaspending_api/search/v2/views/spending_by_category.py

```diff
@@ -265,14 +267,19 @@ def location(self) -> list:
             values = ['pop_congressional_code', 'pop_state_code']
 
         self.queryset = self.common_db_query(filters, values)
+
         # DB hit here
         query_results = list(self.queryset[self.lower_limit:self.upper_limit])
 
         results = alias_response(ALIAS_DICT[self.category], query_results)
         for row in results:
```

Comment:
Is this better than `annotate`-ing the query itself?

Reply:
We're using `alias_response` here b/c `django-mock-queries` doesn't support aliasing columns, so we're doing it this way to keep the code "testable"

Reply:
Works for me.